### PR TITLE
Creates CD Tooling

### PR DESCRIPTION
**Pull request by HaSistrunk**
Sets up application for continuous deployment:

- adds appspec file
- update application configs
- add deploy keys to Travis configs
- improve port settings

### CODE REVIEW

**Review by p-galligan — APPROVED** [review]
Looks good to me.

File: appspec.yml.deploy [hunk 1]
```diff
@@ -0,0 +1,20 @@
+version: 0.0
+os: linux
+files:
+  - source: ./
+    destination: ${ROOT_DIR}
+hooks:
+  ApplicationStop:
+    - location: deploy_scripts/set_permissions.sh
```
Comment (helrond) [inline appspec.yml.deploy:8]: This should be in the AfterInstall step.

**Review by helrond — CHANGES_REQUESTED** [review]
Just one minor thing....

**Review by helrond — APPROVED** [review]

**Review by bonniegee — APPROVED** [review]

File: ursa_major/config.py.deploy [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+SQL_USER = "${SQL_USER}"
+SQL_PASSWORD = "${SQL_PASSWORD}"
+SQL_HOST = "${SQL_HOST}"
+SQL_PORT = ${SQL_PORT}
+
+STORAGE_ROOT = "${STORAGE_ROOT}"
+STORAGE_SRC_DIR = "${STORAGE_SRC_DIR}"
+STORAGE_TMP_DIR = "${STORAGE_TMP_DIR}"
+STORAGE_DEST_DIR = "${STORAGE_DEST_DIR}"
+DERIVATIVE_CREATION_DIR = "{DERIVATIVE_CREATION_DIR}"
```
Comment (bonniegee) [inline ursa_major/config.py.deploy]: @HaSistrunk I think there's a missing `$` here.

**Comment by HaSistrunk** [comment]
I'm also missing a SCHEMA_PATH variable that was in the original config. Trying to figure out how that's used, but assuming it's necessary.

**Comment by bonniegee** [comment]
> I'm also missing a SCHEMA_PATH variable that was in the original config. Trying to figure out how that's used, but assuming it's necessary.

I'm having trouble finding the commit that did this, but I believe this config was removed when we added `rac-schemas` and it just never made its way to deployment.

**Pull request merged**